src/applications/disability-benefits/all-claims/config/form.js

```diff
@@ -289,6 +290,12 @@ const formConfig = {
           uiSchema: summaryOfDisabilities.uiSchema,
           schema: summaryOfDisabilities.schema,
         },
+        conclusionUnemployability: {
+          title: 'Conclusion',
```

Comment:
Anyway we can make the title and path reflect 4192, seems a little too generic.

Reply:
changed!

Reply:
<img width="701" alt="screen shot 2018-11-08 at 2 54 08 pm" src="https://user-images.githubusercontent.com/41440372/48223870-32029700-e366-11e8-8837-5dd5538fc915.png">

Reply:
not sure what's up. Is this not showing up for you? ^^
\